AutoGraph/WebSocketClient.swift

```diff
@@ -184,7 +187,7 @@ open class WebSocketClient {
     
     func sendSubscription(request: SubscriptionRequestSerializable) throws {
         let subscriptionPayload = try request.serializedSubscriptionPayload()
-        guard self.state == .connected else {
+        guard case let .connected(receivedAck) = self.state, receivedAck == true else {
```

Comment:
for the future, i think you can just do `case .connected(receivedAck: true) = self.state